src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,45 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final Object[] keys = new Object[MAX_SIZE];
+    private final Object[] values = new Object[MAX_SIZE];
```

Comment:
1. try to use generic types
2. https://github.com/mate-academy/jv-storage/blob/master/checklist.md#all-non-constant-fields-should-be-initialized-in-the-constructor
```suggestion
    private final K[] keys = new (K[]) Object[MAX_SIZE];
    private final V[] values = new (V[]) Object[MAX_SIZE];
```